### ISSUE TITLE
TypeError: can only concatenate str (not "list") to str 

### ISSUE THREAD

**Issue by shad0w0lf** [Tib3rius/AutoRecon#217]
After starting to scan an ip address the script is receiving the following error: 

`─# python3 autorecon.py 10.10.143.136
[*] Scanning target 10.10.143.136
[*] [10.10.143.136/all-tcp-ports] Discovered open port tcp/80 on 10.10.143.136
[*] [10.10.143.136/all-tcp-ports] Discovered open port tcp/21 on 10.10.143.136
[*] [10.10.143.136/all-tcp-ports] Discovered open port tcp/22 on 10.10.143.136
[*] [10.10.143.136/tcp/80/http/vhost-enum] The target was not a hostname, nor was a hostname provided as an option. Skipping virtual host enumeration.
[*] [10.10.143.136/tcp/80/http/known-security] [tcp/80/http/known-security] There did not appear to be a .well-known/security.txt file in the webroot (/).
[*] [10.10.143.136/tcp/80/http/curl-robots] [tcp/80/http/curl-robots] There did not appear to be a robots.txt file in the webroot (/).
Task exception was never retrieved
future: <Task finished name='Task-4' coro=<start_heartbeat() done, defined at /opt/AutoRecon/autorecon/main.py:129> exception=TypeError('can only concatenate str (not "list") to str')>                                                                                                 
Traceback (most recent call last):                                                                                                                                                                                                                                                       
  File "/opt/AutoRecon/autorecon/main.py", line 161, in start_heartbeat                                                                                                                                                                                                                  
    info('{bgreen}' + current_time + '{rst} - There are {byellow}' + str(count) + '{rst} scans still running against {byellow}' + target.address + '{rst}' + tasks_list)                                                                                                                 
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~                                                                                                                  
TypeError: can only concatenate str (not "list") to str`

I ended it with ctrl+c since it was stucking.

**Comment by oregoncyber** [comment]
having the same issue

**Comment by Tib3rius** [comment]
This should be fixed in 2.0.35.
